src/System.Management.Automation/engine/regex.cs

```diff
@@ -240,7 +240,7 @@ internal static string Escape(string pattern, char[] charsNotToEscape)
                 //
                 // if it is a wildcard char, escape it
                 //
-                if (IsWildcardChar(ch) && !charsNotToEscape.Contains(ch))
+                if ((IsWildcardChar(ch) || ch == '`') && !charsNotToEscape.Contains(ch))
```

Comment:
```suggestion
                if ((IsWildcardChar(ch) || ch == escapeChar) && !charsNotToEscape.Contains(ch))
```

Reply:
@iSazonov I can't remember if this was the actual correct fix to the `Escape` method. We probably need to have a 2nd attempt at fixing public API for this in another PR.

Reply:
Let's discuss in related issue.